core/tests/test_utils.py

```diff
@@ -2174,8 +2175,9 @@ class GenericTestBase(AppEngineTestBase):
     # can override the following class-level constant.
     AUTO_CREATE_DEFAULT_SUPERADMIN_USER: bool = True
 
-    SUPER_ADMIN_EMAIL: Final = 'tmpsuperadmin@example.com'
-    SUPER_ADMIN_USERNAME: Final = 'tmpsuperadm1n'
+    SUPER_ADMIN_EMAIL: Final = (
+        f'tmpsuperadm1n{uuid.uuid4().hex[:6]}@example.com')
+    SUPER_ADMIN_USERNAME: str = f'tmpsuperadm1n{uuid.uuid4().hex[:6]}'
```

Comment:
Updated to allow unique super admins for each setUp(), so tests can be run in concurrency.